cmd/immuclient/immuclient.go

```diff
@@ -87,3 +102,16 @@ func options() *client.Options {
 	}
 	return options
 }
+
+func needsServer(cmd *cobra.Command) bool {
```

Comment:
To avoid to inject client dependency in commands in immuadmin I have put a type commandlineDisc struct{} without dependency. For mangen and version you can simply use the same strategy and avoid to add this method and following map